includes/class-wc-taxjar-integration.php

```diff
@@ -825,6 +831,7 @@ public function calculate_totals( $wc_cart_object ) {
 
 		$address = $this->get_address( $wc_cart_object );
 		$line_items = $this->get_line_items( $wc_cart_object );
+		$shipping_total = $wc_cart_object->get_shipping_total();
```

Comment:
@dallendalton Just changing this to use the `get_shipping_total` helper method instead?

Reply:
Yeah we had to do it the other way for older versions of WooCommerce we no longer support. During all future changes I will be making updates to use the proper ways.